### DISCUSSION TITLE
Is there a way to pass the tags in Publish-AdfV2FromJson

### DISCUSSION THREAD

**Discussion by jrkumar-conga** [Azure-Player/azure.datafactory.tools#234] [Q&A] [answered]
Hello,

In our azure subscription tags are made mandatory thus when we are using _Publish-AdfV2FromJson_ to create the ADF objects we are getting Resource 'adf-mysync-test12345678' was disallowed by policy. Reasons: 'Lifecycle tag is missing','BuildDate tag is missing', etc. 
when DataFactory doesn't exists. Is there a way to pass the tags in Publish-AdfV2FromJson . If not what is the best option to handle this scenario. 

Please let me know your thoughts

Thanks in Advance.

-Raghu

**Comment by NowinskiK** [comment, marked as answer]
No, tags are not supported at the moment. 
If you need tags in your newly created ADF then I suggest to create new ADF instance before run publish cmdlet.
Feel free to create [new issue](https://github.com/SQLPlayer/azure.datafactory.tools/issues/new?assignees=&labels=&template=feature_request.md&title=) about such feature.
